### PR TITLE
PMA-013 Board

### PR DESCRIPTION
**Pull request by AlexanderSUS**
Create board slice
Create board service

### CODE REVIEW

File: src/constants/text.ts [hunk 1]
```diff
@@ -25,3 +25,8 @@ export const langButton = {
   ru: 'ru',
   en: 'en',
 };
+
+export const boardPage = {
+  title: 'You boarads',
```
Comment (Gibr) [inline src/constants/text.ts]: typo, "Your boards"

**Review by Gibr — COMMENTED** [review]
i got this errors
CORS error

File: src/constants/text.ts (hunk 1 above)
Comment (AlexanderSUS) [reply src/constants/text.ts]: Fix typo. Regarding errors in console I'm trying to fix it, but still without result. May be you could help find the solution

File: src/api/boardServise.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { AxiosResponse } from 'axios';
+import api from '.';
+import { Boards, BoardType } from '../types/boards';
+import { ModalFormData } from '../types/modal';
+
+export default class BoardService {
+  static fetchBoards(): Promise<AxiosResponse<Boards>> {
+    return api.get('/boards');
```
Comment (allyksander) [inline src/api/boardServise.ts:8]: We have enum Endpoints.ts (./src/constants/endpoints.ts), we can replace '/boards' in it

**Review by allyksander — APPROVED** [review]

File: src/api/boardServise.ts (hunk 2 above)
Comment (AlexanderSUS) [reply src/api/boardServise.ts:8]: You right, will do this in next PR to be avoid excessive merge

**Pull request merged**